src/lavinmq/log_formatter.cr

```diff
@@ -3,13 +3,27 @@ require "log"
 module LavinMQ
   struct JournalLogFormat < ::Log::StaticFormatter
     def run
+      string "<#{severity_to_priority}> "
```

Comment:
I'm pretty sure this will create an unnecessary string allocation. Better would be
```suggestion
      @io << "<" << severity_to_priority << "> "
```